.travis.yml

```diff
@@ -1,18 +1,25 @@
 sudo: false
 language: d
-d:
-  - dmd-nightly
-  - dmd-beta
-  - dmd
-  - ldc-beta
-  - ldc
 
 os:
   - linux
-  - osx
 
 env:
-  - BUILD=
-  - BUILD=dub
+    matrix:
+        - BUILD=
+        - BUILD=dub
+    global:
+        - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
 
-script: ./.travis.sh
+install: beaver docker build .
+
+script: 
+    - beaver docker run make pkg
```

Comment:
All these should go straight into a separate stage. Please see https://github.com/sociomantic-tsunami/dmqnode/blob/7b75018e43aeebca4eed0e86e6b2d4b8291124da/.travis.yml#L34-L45 for an example...